system-contracts/scripts/calculate-hashes.ts

```
@@ -65,7 +65,7 @@ const getSolidityContractsDetails = (dir: string): ContractDetails[] => {
 const YUL_ARTIFACTS_DIR = "artifacts";
 
 const getYulContractDetails = (dir: string, contractName: string): ContractDetails => {
-  const bytecodePath = join(dir, YUL_ARTIFACTS_DIR, contractName + ".yul.zbin");
+  const bytecodePath = join(dir, YUL_ARTIFACTS_DIR, contractName + ".yul", contractName + ".yul.zbin");
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)